rust/lance-index/src/scalar/inverted.rs

```diff
@@ -41,8 +60,13 @@ pub struct InvertedIndex {
 }
 
 impl InvertedIndex {
+    pub fn new() -> Self {
+        Self::default()
+    }
+
     // map tokens to token ids
     // ignore tokens that are not in the index cause they won't contribute to the search
+    #[instrument(level = "debug")]
```

Comment:
Let's add `skip_all` (My default is to add `skip_all` to everything as I rarely find the extra info useful and it becomes a refactor risk when someone adds a parameter without thinking in the future)?